geomesa-gs-wfs/src/main/scala/org/geomesa/gs/wfs/output/ArrowOutputFormat.scala

```diff
@@ -15,20 +15,21 @@ import org.geoserver.ows.Response
 import org.geoserver.platform.Operation
 import org.geoserver.wfs.WFSGetFeatureOutputFormat
 import org.geoserver.wfs.request.{FeatureCollectionResponse, GetFeatureRequest}
+import org.geotools.api.feature.simple.{SimpleFeature, SimpleFeatureType}
 import org.geotools.api.filter.sort.SortOrder
 import org.geotools.data.simple.SimpleFeatureCollection
 import org.geotools.util.factory.Hints
-import org.locationtech.geomesa.arrow.{ArrowEncodedSft, ArrowProperties}
-import org.locationtech.geomesa.arrow.io.FormatVersion
+import org.locationtech.geomesa.arrow.io.{FormatVersion, SimpleFeatureArrowFileReader}
 import org.locationtech.geomesa.arrow.vector.SimpleFeatureVector.SimpleFeatureEncoding
+import org.locationtech.geomesa.arrow.{ArrowEncodedSft, ArrowProperties}
 import org.locationtech.geomesa.index.conf.QueryHints._
 import org.locationtech.geomesa.process.transform.ArrowConversionProcess.ArrowVisitor
 import org.locationtech.geomesa.utils.collection.CloseableIterator
 import org.locationtech.geomesa.utils.geotools.SimpleFeatureTypes
 import org.locationtech.geomesa.utils.io.WithClose
 
-import java.io.{BufferedOutputStream, OutputStream}
-import scala.collection.JavaConverters._
+import java.io.{BufferedOutputStream, ByteArrayInputStream, OutputStream}
+import scala.jdk.CollectionConverters._
```

Comment:
CI is currently busted due to eclipse infra issues, but i believe `scala.jdk.CollectionConverters._` isn't available in 2.12 and this needs to be reverted back to `scala.collection.JavaConverters._` (which is in both 2.12 and 2.13)